src/main/java/org/neo4j/opencypherdsl/Aliased.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
let's not add opencypher-dsl as code but as dependency, release is meant to go out this week.

Reply:
can you trigger @michael-simons to review + merge https://github.com/neo4j/sdn-rx/pull/244?